src/IO/GEHDF5Wrapper.cxx

```diff
@@ -168,6 +189,7 @@ Succeeded GEHDF5Wrapper::initialise_scanner_from_HDF5()
     int num_detectors_per_ring = num_transaxial_blocks_per_bucket*num_transaxial_crystals_per_block*radial_modules_per_system;
     int default_num_arccorrected_bins = max_num_non_arccorrected_bins;
     float inner_ring_radius = physical_scanner_radius;
+    //AB TODO check if following is valid
```

Comment:
hmmm. well-spotted.

@pw351. questions for you:

- we get `physical_scanner_radius` from `/HeaderData/SystemGeometry/sourceRadius` above, but that seems incorrect. (it probably has to do with the good old days when they had a line source inside the scanner, although I'm not sure).
- `inner_ring_diameter` is set from `/HeaderData/SystemGeometry/effectiveRingDiameter`, so this variable has the wrong name.

So, although I agree with the formulas, I wonder if the input is correct. What are the corresponding values?

Note however that STIR will currently only use its `ring_radius+DOI`, resulting in `effective_ring_radius`, which will be correct.

Reply:
Yes, this part of code is basically not used yet, as far as I remember! 
Looking at the values for :
/HeaderData/SystemGeometry/sourceRadius: 311.3
/HeaderData/SystemGeometry/effectiveRingDiameter: 640.6

Indeed these are incorrect!
I will correct these!

Reply:
`effectiveRingRadius` would therefore be 320.3,  it doesn't make the `sourceRadius` entirely unbelievable to mean "inner ring radius", but just a bit too small.

If the RDF file doesn't give inner ring radius elsewhere, there doesn't seem to be a way for us to figure out what it is. I then suggest it to `effectiveRingRadius - 7mm` using a (somewhat arbitrary) DOI for LYSO (as I believe we have for other GE scanners). Of course, for the Discovery IQ, the crystal is BGO, so therefore its DOI would be smaller. I guess they don't record the scintillator material, so this seems all to lead to some ugly hacks.

Reply:
Basically, the physical ring radius must be 311.8 precisely.  The depth of interaction can be calculated using effectiveRingRadius - physicalRingRadius which in my case is 320.3-311.8 which works out to be 8.5. I also need this to be fixed in scanner.cxx. I will do this ASAP.

Reply:
is the 311.8 somewhere in the header? otherwise we have to hard-code DOI somehow (if it's for scanner where we don't know it)